src/Peering/Peering/Direct/SetAzureDirectPeeringConnectionCommand.cs

```diff
@@ -215,13 +215,13 @@ private PSDirectConnection UpdateMD5Authentication()
         /// </returns>
         private PSDirectConnection UpdateIpV4Prefix()
         {
-            if (this.InputObject is PSDirectConnection inputObject)
+            if (this.InputObject.GetType().Equals(typeof(PSDirectConnection)))
```

Comment:
Why use direct type comparsion instead of `is`? Except for extremely rare cases, `is` is what you want.